docs/tutorials/build-files-archive-with-dal/get-dal-params.md

```diff
@@ -167,25 +173,26 @@ cp target/wasm32-unknown-unknown/release/files_archive.wasm .
 smart-rollup-installer get-reveal-installer -P _rollup_node/wasm_2_0_0 \
   -u files_archive.wasm -o installer.hex
 
-octez-client --endpoint ${ENDPOINT} \
-  originate smart rollup files_archive from "${alias}" of kind wasm_2_0_0 \
+octez-client originate smart rollup files_archive from "${alias}" of kind wasm_2_0_0 \
   of type unit with kernel "$(cat installer.hex)" --burn-cap 2.0 --force
 
-octez-smart-rollup-node --endpoint ${ENDPOINT} \
+octez-smart-rollup-node --endpoint http://127.0.0.1:8732  \
   run observer for files_archive with operators --data-dir _rollup_node \
   --dal-node http://localhost:10732 --log-kernel-debug
 ```
 
-To use it, save it in a file with an `sh` extension, such as `deploy_smart_rollup.sh` and give it executable permission.
+To use it, save it in a file with an `sh` extension, such as `deploy_smart_rollup.sh` and give it executable permission by running `chmod +x deploy_smart_rollup.sh`.
 Then you can run it any tme you update the `lib.rs` or `Cargo.toml` files to deploy a new Smart Rollup by passing your account alias, as in this example:
 
 ```bash
-./deploy_smart_rollup.sh $MY_ACCOUNT
+./deploy_smart_rollup.sh my_wallet
 ```
 
+This script assumes that your local node is running at http://127.0.0.1:8732.
```

Comment:
I think the `octez-client` command above also requires this assumption. I suggest you move it before the first command that requires it.

Reply:
Added this:

>   The Octez client assumes that your local node is running at http://127.0.0.1:8732. If your node is running at a different host name or port, pass the host name and port of the node to the `--endpoint` argument. For example, if the node is running on port 8733, include `--endpoint http://127.0.0.1:8733` in the command.